### ISSUE TITLE
Complete noob, trying to get this to work. 

### ISSUE THREAD

**Issue by snvoid** [JuanuSt/mmega#3]
Hey @JuanuSt 

I'm a noob trying to figure out how to get this to work. I keep running into a command not found error and i'm not sure what i'm doing wrong. Any help/guidance would be appreciated. I changed lines 94/95 to point to the bins, but i can't seem to get it to work. 

Please see attachment for pic. Thanks. 

![helpanoob](https://user-images.githubusercontent.com/60519765/74557091-70094600-4f2d-11ea-86e4-1ab2dff0cb09.png)

**Comment by JuanuSt** [comment]
Hi snvoid,

Thanks for reporting the bug.

I just test the old script, and it still works. Let's forget the encryption for the moment, just to see if it works in plaintext.

I'm sorry, it was my fault, but the scritps, normal and cron in the repository, have no permission to be executed. So you have to allow their execution with the command:

``
chmod +x mmega.sh
``

Indeed this script has no installer, so, when download it, you have to go the downloaded directory and execute the script:

``cd /path/to/mmega/directory
``
``./mmega.sh -c accounts.txt -m df
``

There are another solutions, one is to add the directory path to the PATH, so it can be found:

``
PATH=/path/to/mmega/directory:$PATH
``

The other one is to create an bash alias to it. You can create in you home directory the file .bash_aliases and insert this alias:

``alias mmega="/path/to/directory/mmega.sh"
``

I think that simply granting the execution permissions and using the dot-slash (./mmega.sh) before the script should make it work.

Let me know if it worked.

**Comment by snvoid** [comment]
Thank you so much!!! I just tried it the way you said and it worked perfectly this time! This tool is a life saver. Thank you so much for helping me with this! I was scratching my head trying to figure out what I was doing wrong. Thanks for explaining it to me. After using your instructions and trying it using the plain text, I went ahead and tried it using the .gpg file and that worked as well!!! 

Thank you again! I went ahead and closed the ticket/issue. 

👍 😃
